test/test-network/systemd-networkd-tests.py

```diff
@@ -4468,9 +4468,9 @@ def test_dhcp_client_ipv4_use_routes_gateway(self):
     def _test_dhcp_client_ipv4_use_routes_gateway(self, use_routes, use_gateway, dns_and_ntp_routes, classless):
         testunit = 'dhcp-client-ipv4-use-routes-use-gateway.network'
         testunits = ['25-veth.netdev', 'dhcp-server-veth-peer.network', testunit]
-        testunits.append(f'{testunit}.d/use-routes-{use_routes}.conf');
```

Comment:
"tell me this Python was written by a C developer without telling me it was written by a C developer" :-D